powerauth-push-client/src/main/java/io/getlime/push/client/PushServerClient.java

```diff
@@ -138,23 +144,48 @@ public boolean createDevice(String appId, String token, MobilePlatform platform,
             throw new PushServerClientException(error);
         }
 
-        logger.info("Calling create device service, appId: {}, token: {}, platform: {} - start", appId, maskToken(token), platform);
+        logger.info("call={}, callType={}, action: createDevice, state: initiated, appId: {}, token: {}, platform: {}, activationId: {}",
+                "/push/device/create", "POST", appId, maskToken(token), platform, activationId);
+        Response response = postObjectImpl("/push/device/create", new ObjectRequest<>(request));
+        logger.info("call={}, callType={}, action: createDevice, state: succeeded", "/push/device/create", "POST");
+
+        return response.getStatus().equals(Response.Status.OK);
+    }
+
+    /**
+     * Register a device to the push server.
+     *
+     * @param request Create device request.
+     * @return True if device registration was successful, false otherwise.
+     * @throws PushServerClientException In case of network, response / JSON processing, or other IO error.
+     */
+    public boolean createDevice(final CreateDeviceRequest request) throws PushServerClientException {
+        String error = CreateDeviceRequestValidator.validate(request);
+        if (error != null) {
+            throw new PushServerClientException(error);
+        }
+
+        logger.info("call={}, callType={}, action: createDevice, state: initiated, appId: {}, token: {}, platform: {}, environment: {}",
+                "/push/device/create", "POST", request.getAppId(), maskToken(request.getToken()), request.getPlatform(), request.getEnvironment());
         Response response = postObjectImpl("/push/device/create", new ObjectRequest<>(request));
-        logger.info("Calling create device service, appId: {}, token: {}, platform: {} - finish", appId, maskToken(token), platform);
+        logger.info("call={}, callType={}, action: createDevice, state: succeeded", "/push/device/create", "POST");
```

Comment:
🤔 Is parametrizing the log message by the static string value needed?

Reply:
I am following the previous feedback in this PR.